web/css/source/_theme.less

```diff
@@ -324,6 +370,20 @@ h1.site-intro{
   }
 }
 
+// hackity hack hack - Maganto style! - overriding Luma one ID at a time as I couldn't get to work any other way
+a#ui-id-1.level-top.ui-corner-all,
+a#ui-id-2.level-top.ui-corner-all,
+a#ui-id-3.level-top.ui-corner-all,
+a#ui-id-4.level-top.ui-corner-all,
+a#ui-id-5.level-top.ui-corner-all,
+a#ui-id-6.level-top.ui-corner-all,
+a#ui-id-7.level-top.ui-corner-all,
+a#ui-id-8.level-top.ui-corner-all
+a#ui-id-9.level-top.ui-corner-all,
+a#ui-id-10.level-top.ui-corner-all{
+  color: white;
```

Comment:
I reckon by looking at the relevant rule in the luma theme, we can do something here that is more specific and will override correctly. Failing that I think this is probably better, simply because it is not limited to 10 items:

```
a.level-top {
  color: white !important;
} 
```